Dockerfile

```diff
@@ -102,7 +102,12 @@ RUN \
     ubuntu-drivers-common \
     xxd \
     zip \
-    zlib1g-dev && \
+    zlib1g-dev \
+    cuda-toolkit-12-6 \
+    libnvidia-compute-550-server \
+    libfdt-dev \
+    protobuf-compiler-grpc \
+    bazel-bootstrap && \
```

Comment:
Some of the apt packages are listed already (in alphanumeric order) - please don't duplicate them.

Reply:
Also, the `protobuf-compiler-grpc` is part of the grpc source repository build so this is redundant.

Reply:
sure, fixed